server/views/river-and-sea-levels.html

```diff
@@ -50,88 +50,60 @@ <h2 class="govuk-heading-m govuk-!-margin-top-7">
     </div>
   </div>
 {% else %}
-  {% if model.isMultilpleMatch %}
+  {% if model.stations | length %}
```

Comment:
It's not clear from the diff but i've just removed the code from the inside the isMultipleMatch conditional code block as multiple matches are now handled by rivers-and-sea-level-list.html so the code block was no longer reachable. 

The removed code block contained a reference to river_id which either needed fixing or the code block removing.